be/src/vec/exprs/vexpr.cpp

```diff
@@ -97,11 +97,11 @@ TExprNode create_texpr_node_from(const void* data, const PrimitiveType& type, in
         break;
     }
     case TYPE_DATEV2: {
```

Comment:
TODOs:
1. verify  datev1 and datetimev1 have precision ？？？
2. table1 is datev2 and table2 is datev2 ， generate runtime filter。
3. table1 is datetimev2 and table2 is datetimev2 ， generate runtime filter。